counterpartylib/lib/blocks.py

```diff
@@ -93,8 +93,8 @@ def parse_tx(db, tx):
         order.parse(db, tx, message)
     elif message_type_id == btcpay.ID:
         btcpay.parse(db, tx, message)
-    elif message_type_id == issuance.ID:
-        issuance.parse(db, tx, message)
+    elif message_type_id == issuance.ID or message_type_id == issuance.SUBASSET_ID:
```

Comment:
Yeah, I originally wrote it with 2 `elif`s.  But then I changed it to stand out and be obvious that both of these messages are routed to the same function.  Just a preference.